include/libopencm3/stm32/lptimer.h

```diff
@@ -0,0 +1,36 @@
+/* This provides unification of code over STM32 subfamilies */
+
+/*
+ * This file is part of the libopencm3 project.
+ *
+ * Copyright (C) 2019 Guillaume Revaillot <g.revaillot@gmail.com>
+ *
+ * This library is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this library.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <libopencm3/cm3/common.h>
+#include <libopencm3/stm32/memorymap.h>
+
+#if defined(STM32F4)
+#       include <libopencm3/stm32/f4/lptimer.h>
+#elif defined(STM32L0)
+#       include <libopencm3/stm32/l0/lptimer.h>
+#elif defined(STM32l4)
+#       include <libopencm3/stm32/l4/lptimer.h>
+#elif defined(STM32G0)
+#       include <libopencm3/stm32/g0/lptimer.h>
+#else
```

Comment:
(I'm not going to get upset about it, but this commit is first, and includes these l4/g0 files that don't exist until the subsequent commit, if you want perfect style bonus points, you enable them here as you add the files)